cmd/lakectl/cmd/local.go

```diff
@@ -137,9 +137,12 @@ func warnOnCaseInsensitiveDirectory(path string) {
 }
 
 func buildLocalConfig(syncFlags local.SyncFlags, cfg *Configuration) local.Config {
+	if cfg.Server.Retries.MaxAttempts < 0 {
+		Die("Invalid configuration: max download retries must be non-negative", 1)
+	}
```

Comment:
Please don't - why don't we just change the value to uint?

Reply:
It doesn't really change the user experience (not a breaking change since user needs to provide a non-negative value anyways) so I much prefer it than adding a panic check and nolint comment. It's easier and safer to cast uint to int